src/main.js

```diff
@@ -24,7 +28,9 @@ Vue.use(VueFire);
 /* eslint-disable no-new */
 new Vue({
   router,
+  store,
   el: '#app',
   template: '<App/>',
-  components: { App },
+  render: h => h(App),
```

Comment:
Dobre pytanie, pamietam, ze mialo to chyba znaczenie przy testach jednostkowych. Wracam do poprzedniej wersji ale zostawiam "render: h => h(App)" zakomentowane jakbym jeszcze tego potrzebowal